lib/tic_tac_toe/web/router.ex

```diff
@@ -1,62 +1,54 @@
 defmodule TicTacToe.Web.Router do
   use Plug.Router
-  import TicTacToe.Web.GameSessionPlug
-  alias TicTacToe.Web.GameSessionPlug
+  alias TicTacToe.Web.GameStateStringifier
   alias TicTacToe.Web.View
 
-  @secret String.duplicate("abcdef0123456789", 8)
-
-  plug Plug.Session, store: :cookie, key: "_my_app_session",
-                     encryption_salt: "cookie store encryption salt",
-                     signing_salt: "cookie store signing salt",
-                     key_length: 64, log: :debug
-
   plug Plug.Parsers, parsers: [:urlencoded, :multipart]
   plug Plug.Static, at: "/public", from: :elixir_web_ttt
-  plug :put_secret_key_base
   plug Plug.Logger
   plug :match
-  plug :create_or_find_game
   plug :dispatch
 
-  get ("/tictactoe/options") do
-    response_body = View.game_options()
+  get ("/ttt/options") do
+    response_body = View.stringified_game_options()
     conn |> put_resp_content_type("html") |> resp(200, response_body)
   end
 
-  get ("/tictactoe/play") do
-    game_state = conn |> GameSessionPlug.get_game_state()
-    response_body = View.render_game(game_state)
+  get ("ttt/play/:stringy_game_state") do
+    game_state = GameStateStringifier.parse(stringy_game_state)
+    response_body = View.render_game(game_state, stringy_game_state)
     conn |> put_resp_content_type("html") |> resp(200, response_body)
```

Comment:
I like that you've extracted most of the logic out of this router. Keeps things nice and readable, and ensures this is the boundary to your application and the rest of the logic is nested below.